### ISSUE TITLE
None of the ETKDG versions enforces chirality specified in the molecule SMILES

### ISSUE THREAD

**Issue by ignatovmg** [rdkit/rdkit#5283]
**Describe the bug**
The following molecule should have the 22-23 double bond in trans configuration, however all sampling methods `ETKDG`, `ETKDGv2` and `ETKDGv3` produce conformers with both correct and wrong chirality.

`mol = Chem.MolFromSmiles('Cc3nn(CC(=O)N2CCN(c1ccccc1)CC2)c(C)c3/N=N\c6ccc(CNC(=O)CCC(=O)Nc4cccc5C(=O)NCc45)cc6')`

![image](https://user-images.githubusercontent.com/11196230/168115006-85c3dbe6-970e-4c53-9542-06f0f0f30e51.png)

**To Reproduce**

Lets create a molecule check if chirality is parsed
```
mol = Chem.MolFromSmiles('Cc3nn(CC(=O)N2CCN(c1ccccc1)CC2)c(C)c3/N=N/c6ccc(CNC(=O)CCC(=O)Nc4cccc5C(=O)NCc45)cc6')
b = mol.GetBondBetweenAtoms(22, 23)
print(b.GetBeginAtomIdx(),b.GetEndAtomIdx(), b.GetBondType(),b.GetStereo())
```

For some reason we get `STEREOE`, but after running `FindMolChiralCenters` it becomes `STEREOTRANS`, which I guess is correct (not sure what `STEREOE` means):

```
Chem.FindMolChiralCenters(mol, force=True, includeUnassigned=True, useLegacyImplementation=False)
b = mol.GetBondBetweenAtoms(22, 23)
print(b.GetBeginAtomIdx(),b.GetEndAtomIdx(), b.GetBondType(),b.GetStereo())
```

Now it's `STEREOTRANS`. Next I'm trying to run conformer generation with different parameters, the molecule is large so I try setting `useRandomCoords = True` as well, following [this](https://sourceforge.net/p/rdkit/mailman/message/36899701/) topic.

```
seed = 123
counter = 0
for random in [True, False]:
    for etkdg_version in [AllChem.ETKDG, AllChem.ETKDGv2, AllChem.ETKDGv3]:
        mol = Chem.MolFromSmiles('Cc3nn(CC(=O)N2CCN(c1ccccc1)CC2)c(C)c3/N=N/c6ccc(CNC(=O)CCC(=O)Nc4cccc5C(=O)NCc45)cc6')
        Chem.FindMolChiralCenters(mol, force=True, includeUnassigned=True, useLegacyImplementation=False)
        mol = AllChem.AddHs(mol)
        ps = etkdg_version()
        ps.randomSeed = seed
        ps.useRandomCoords = random
        ps.numThreads = 32
        ps.enforceChirality = True
        cid_list = AllChem.EmbedMultipleConfs(mol, 100, ps)
        
        sdf_file = 'etkdg_test/' + str(counter) + '.sdf'
        writer = Chem.SDWriter(sdf_file)
        for cid in cid_list:
            writer.write(mol, confId=cid)
        writer.close()
        
        counter += 1
```

Regardless of the method or whether i use `Chem.FindMolChiralCenters` I get both cis and trans conformers in the output. Here is an example of a wrong conformer in the generated samples:

<img width="400" alt="image" src="https://user-images.githubusercontent.com/11196230/168118860-85410553-d536-40b5-91d9-232b26c792ec.png">

**Expected behavior**
Generated conformers should contain only trans isomers.

**On a side note**
If the chirality is specified in SMILES why do we need (do we?) to run `FindMolChiralCenters` or `FindPotentialStereo` (following the [cookbook](https://www.rdkit.org/docs/Cookbook.html#stereochemistry))

**Configuration (please complete the following information):**
 - RDKit version: 2022.03.2
 - OS: Ubuntu 18.04.1
 - Python version (if relevant): 3.7.13
 - Are you using conda? yes
 - If you are using conda, which channel did you install the rdkit from? conda-forge

**Comment by greglandrum** [comment]
@ignatovmg : sorry I never replied to this one. 
This is a known problem and will be fixed in an upcoming RDKit release

**Comment by greglandrum** [comment]
Fixed in #5967 but the automatic connection didn't get made
